src/translations/po/de.po

```diff
@@ -0,0 +1,247 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
can you please add yourself to the copyright header? e.g.

```
# SPDX-License-Identifier: GPL-3.0-or-later
#
# SPDX-FileCopyrightText: 2021 name <email>
```

name can be a pseudonym if you're concerned about privacy. the most important part is email.